spec/turn_spec.rb

```diff
@@ -0,0 +1,59 @@
+require './lib/turn'
+require './lib/card'
+require 'rspec'
+
+describe Turn do
+  it "exists" do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    expect(turn).to be_a(Turn)
+    expect(card).to be_a(Card)
+
+  end
+
+  it "has attributes" do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    expect(turn.guess).to eq "Juneau"
+    expect(turn.card).to eq card
+
+  end
+
+  it "it checks if the answer's correct" do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
```

Comment:
This test is failing because the answer you are passing in has a capital J but the `correct?` method in your `Turn` class does a `downcase` on `card.answer` so they do not match. You can either update your test to pass in a lower case answer ("juneau") or update your `correct?` method to also call `downcase` on `guess`.

Reply:
This is also causing a bunch of tests for `Round` to fail. Addressing the case-sensitive aspect of the guess will resolve all of them though 👍🏻